xmake/modules/package/tools/scons.lua

```diff
@@ -0,0 +1,36 @@
+--!A cross-platform build utility based on Lua
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+-- Copyright (C) 2015-present, TBOOX Open Source Group.
+--
+-- @author      PucklaMotzer09
+-- @file        scons.lua
+--
+
+-- imports
+import("core.base.option")
+import("lib.detect.find_tool")
+
+-- build package
+function build(package, configs, opt)
+    opt = opt or {}
+    local buildir = opt.buildir or os.curdir()
+    local njob = opt.jobs or option.get("jobs") or tostring(math.ceil(os.cpuinfo().ncpu * 3 / 2))
+    local scons = assert(find_tool("scons"), "scons not found!")
+    local argv = {"-C", buildir, "-j", njob}
```

Comment:
Can you add verbose output?

```lua
if option.get("verbose") then
    table.insert("argv", ...) --  scons VERBOSE=1?
end
```

Reply:
I looked at the output of ```scons -H``` and there seems to be no such thing as verbose output. ```-v``` is used for version displaying. At default scons outputs every invoked compiler call.

Reply:
try `scons VERBOSE=1`?

https://scons.org/doc/1.0.1/HTML/scons-user/x1884.html

Reply:
This does not seem to change anything. I also see every compiler call. But we could leave it in anyway.

Reply:
ok, I merged it first. we can improve it later.